cranelift/codegen/src/isa/unwind/systemv.rs

```diff
@@ -0,0 +1,129 @@
+//! System V ABI unwind information.
+
+use alloc::vec::Vec;
+use gimli::write::{Address, FrameDescriptionEntry};
+use thiserror::Error;
+
+#[cfg(feature = "enable-serde")]
+use serde::{Deserialize, Serialize};
+
+type Register = u16;
+type Expression = Vec<u8>;
```

Comment:
FYI, `gimli::write::Expression` is no longer a simple `Vec<u8>` after https://github.com/gimli-rs/gimli/pull/479, so this isn't going to work.

It seems like you don't actually need `impl From<gimli::write::CallFrameInstruction> for CallFrameInstruction` though, so it would be possible to simply delete all the `Expression` support. If you do need to support expressions, then you'll need to come up with your own serializable version of them. gimli's expressions are used for other debug sections too, thus they potentially have references to DIEs and can't be individually serialized.

Reply:
See https://github.com/philipc/wasmtime/commit/103d68d6c5e6201f76a17b14b29ddca82ad02350 for the changes I think are needed. I can open a PR once a new gimli version is released.

Reply:
@philipc thanks for the heads up!  When we update gimli, I think we can work around this by removing expression support from this representation since we don't need it to describe our frames currently.